src/DiIiS-NA/D3-GameServer/GSSystem/PowerSystem/Implementations/HeroSkills/Wizard.cs

```diff
@@ -1292,30 +1292,33 @@ public override IEnumerable<TickTimer> Main()
 
 				if (Rune_B > 0)     //Raging Storm
 				{
-					var twisters = Twister.GetActorsInRange<EffectActor>(5f).Where(i => ((i.SNO == ActorSno._wizard_tornado) && (i != Twister)));
-					if (twisters.Count() > 0)
```

Comment:
Here we filtered the first time by `Count()` and the second time filtered by iteration using foreach